### PR TITLE
Create Get-SPOTenantRenameSitePrioritization.md

### PR DESCRIPTION
**Pull request by AnfraMsft**
[PubOps PR review comments](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/613#issuecomment-2040107457)
***

Introduced documentation for Get cmdlet for site prioritization

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## SYNOPSIS
+Returns the list of sites that are prioritized for early execution, as part of Advanced Tenant Rename.
+## SYNTAX
+```
+Get-SPOTenantRenameSitePrioritization [<CommonParameters>]
+```
+## DESCRIPTION
+This cmdlet can be used to view the currently prioritized sites, as part of Advanced Tenant Rename. 
+
+The output is a list of all site URLs, one per row. If desired, the output can be downloaded using the standard PowerShell Out-File option. If you want just the count of prioritized sites, you can use the standard Measure option.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md:25]: >Measure [](http://example.com/codeflow?start=214&length=7)

Measure-Object

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+You must be a SharePoint or Global Administrator to run this cmdlet.
+## EXAMPLES
+### Example 1
+```powershell
+Get-SPOTenantRenameSitePrioritization
+```
+This example will return the list of sites prioritized for the Advanced Tenant Rename scheduled in the tenant, if one exists. 
+### Example 2
+```powershell
+Get-SPOTenantRenameSitePrioritization | measure
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md:36]: >measure [](http://example.com/codeflow?start=40&length=7)

Measure-Object

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+```
+This example will return the count of prioritized sites.
+### Example 3
+```powershell
+Get-SPOTenantRenameSitePrioritization | Out-File -FilePath <path>
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md:41]: ><path> [](http://example.com/codeflow?start=59&length=6)

.\sites.txt

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+```
+This example will download the list of prioritized sites to the specified path.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md:43]: >to the specified path [](http://example.com/codeflow?start=57&length=21)

and saves it in file 'sites.txt' in the current directory.

**Review by samkabue — APPROVED** [review]
:shipit:

**Comment by Stacyrch140** [comment]
Hi @AnfraMsft - 

**Following are blocking issues that need to be resolved before we can merge:**

sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md
- Add this file to its corresponding TOC, and include the TOC with this PR. 


***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=main)

**Blocking**
- When a new article is added, the same PR must also contain at least one updated TOC with an entry for the new article. Exceptions are allowed when the file has ROBOTS: NOINDEX, NOFOLLOW in the metadata.

***

After you resolve the blocking issues, remove the **Back to submitter** label and add the **Sign off** label again. 

If you need help resolving issues in this PR, post your question in the [Ask an Admin Teams Channel](https://teams.microsoft.com/l/channel/19%3aea9a561f603c4fbb9cf3931a6849eae4%40thread.skype/Ask%2520an%2520Admin?groupId=7079ffbf-1420-4ee2-9ddc-a2c351337320&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).


Thanks, 
Stacy

**Comment by Stacyrch140** [comment]
> Introduced documentation for Get cmdlet for site prioritization

**Comment by maggierui** [comment]
Simliar to the other PR, this PR requester has already added the file into the TOC https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/614#issuecomment-2048187234

**Comment by Stacyrch140** [comment]
Hi @AnfraMsft  - 

**Following is a blocking issue that needs to be resolved before we can merge:**

sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRenameSitePrioritization.md
- Does this need to be added to a TOC? If so, add this article to a TOC and include the TOC with this PR.

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=main)

**Blocking**
- When a new article is added, the same PR must also contain at least one updated TOC with an entry for the new article. Exceptions are allowed when the file has ROBOTS: NOINDEX, NOFOLLOW in the metadata.

***

After you resolve the blocking issues, remove the **Back to submitter** label and add the **Sign off** label again. 

If you need help resolving issues in this PR, post your question in the [Ask an Admin Teams Channel](https://teams.microsoft.com/l/channel/19%3aea9a561f603c4fbb9cf3931a6849eae4%40thread.skype/Ask%2520an%2520Admin?groupId=7079ffbf-1420-4ee2-9ddc-a2c351337320&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).


Thanks, 
Stacy

**Pull request merged**